development/README.md

```diff
@@ -51,23 +51,46 @@ To check if the installation is successful, perform the following steps:
 `kubectl get ingress ing`
 Copy the address of the ingress `ing` and open it in a browser to display the Prow status on the dashboard.
 
-## Configuration
+## Development
 [Configure Webhook](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md#add-the-webhook-to-github) to enable sending Events from a GitHub repository to Prow.
-You can configure Prow by specifying the `plugins.yaml` and `config.yaml` files. To generate them, use `./development/generate.sh`. The `generate.sh` script combines the `plugins.yaml.tpl` and `config.yaml.tpl` template files with actual values provided as a JSON file and generates output to the `plugins.yaml` and `config.yaml` files. The following snippet is an exmample of the JSON file:
+You can configure Prow by specifying the `plugins.yaml` and `config.yaml` files. 
 
-```
-{
-  "OrganizationOrUser":"yourgithubuser"
-}
-```
+To check if the `plugins.yaml` and `config.yaml` configuration files are correct, run the `check.sh` script.
+In case of changes in the plugins configuration, use the `update-plugins.sh` to apply changes on a cluster.
+In case of changes in the jobs configuration, use the `update-config.sh` to apply changes on a cluster.
 
-Store the updated `plugins.yaml` and `config.yaml` files, which are available for a cluster, in the `prow` directory.
+### Strategy for organising jobs
 
->**NOTE:** You can provide a path to the JSON file from the console input or by specifying the `INPUT_JSON` environment variable.
+The `test-infra` repository has defined configurations for the Prow cluster in the `prow` subdirectory. This directory has the following structure:
+- `cluster` directory, which contains all `yaml` files for `prow` cluster provisioning
```

Comment:
Shouldn't we also use `Prow` instead of `` `prow` `` here?

Reply:
Done